migrations/versions/0322_jobs_add_api_key.py

```diff
@@ -0,0 +1,39 @@
+"""
+
+Revision ID: 0322_jobs_add_api_key
+Revises: 0321_daily_limit_updated
+Create Date: 2021-06-07 13:37:42
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+revision = "0322_jobs_add_api_key"
+down_revision = "0321_daily_limit_updated"
+
+
+def upgrade():
```

Comment:
We had a `created_by_id` for jobs, which makes sense when users upload spreadsheets through the UI but for bulk jobs from the API we are interested in the API key used